### ISSUE TITLE
Sushi-Scan not working on J2K only

### ISSUE THREAD

**Issue by Sapphire-e** [Jays2Kings/tachiyomiJ2K#1572]
### Steps to reproduce

1) Install SushiScan extension
2) Try to use it

### Expected behavior

Works normally

### Actual behavior

Cloudfare ban error

### TachiyomiJ2K version

1.6.6

### Android version

13

### Device

Oppo Find X3 Neo 5G

### Other details

I tried adding a cloudfare interceptor to the extension, and it didn't change anything. (I tried other things so that's why I don't open the issue in the extensions repository.)
The extension works properly on base tachiyomi as well, just not J2K.
Many other users in the Tachiyomi discord server have issues as well.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Tachiyomi](https://github.com/tachiyomiorg/tachiyomi)
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.6.6](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have updated all installed extensions.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by Sapphire-e** [comment]
Ignore this. It seems that someone on the discord also got blocked on base tachiyomi, although a friend and I didn't.
